File: 151-reverse-words-in-a-string/reverse-words-in-a-string.js
/**
 * @param {string} s
 * @return {string}
 */
var reverseWords = function(s) {

    let word = s.trim().split(/\s+/);

    let reversed = "";

    

    for(let i=word.length-1; i>=0; i--){

        reversed = reversed + word[i] + " ";

        
    }

    
    return reversed.trim();
    
};

File: 151-reverse-words-in-a-string/reverse-words-in-a-string.java
class Solution {
    public String reverseWords(String s) {

        String [] word = s.trim().split("\\s+");

        String reversed = "";

        for (int i = word.length - 1; i >= 0; i--) {

            reversed = reversed + word[i] + " ";
        }

        return reversed.trim();

    }
}